projects/libyaml/libyaml_emitter_fuzzer.c

```diff
@@ -296,7 +293,12 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
     if (!yaml_parser_initialize(&parser))
       return 0;
 
-    yaml_parser_set_input_string(&parser, buffer, written);
+    if (!out.buf) {
```

Comment:
i think you should not need this check, if there is no data, yaml_parser_set_input_string should jut parse a 0byte string